.drone.yml

```diff
@@ -77,6 +77,8 @@ pipeline:
     group: parallel
     <<: *sbttestenv
     commands:
+      - rm /dev/random
+      - mknod /dev/random c 1 9
```

Comment:
should't this be /dev/urandom?

Reply:
Nope, we're replacing /dev/random with the same thing that /dev/urandom normally resolves to